test/test_KubeLibrary.py

```diff
@@ -678,3 +694,53 @@ def test_get_cron_job_details_in_namespace(self, mock_lnp):
         kl = KubeLibrary(kube_config='test/resources/k3d')
         cron_job_details = kl.get_cron_job_details_in_namespace('hello', 'default')
         self.assertEqual('mytestlabel', cron_job_details.items.metadata.labels.TestLabel)
+
+    @mock.patch('kubernetes.client.RbacAuthorizationV1Api.delete_namespaced_role')
+    def test_delete_role_in_namespace(self, mock_lnp):
+        mock_lnp.side_effect = mock_delete_role_in_namespace
+        kl = KubeLibrary(kube_config='test/resources/k3d')
+        role_list = kl.delete_role_in_namespace('pod-reader', 'default')
+        self.assertEqual(['pod-reader'], [item.metadata.name for item in role_list.items])
+
+    @mock.patch('kubernetes.client.RbacAuthorizationV1Api.create_namespaced_role')
+    def test_create_role_in_namespace(self, mock_lnp):
+        mock_lnp.side_effect = mock_create_role_in_namespace
+        kl = KubeLibrary(kube_config='test/resources/k3d')
+        name = 'pod-reader'
+        role_manifest = {
```

Comment:
I believe this test has no value, we are not testing anything as everything is mocked. One thing that might be useful is testing handling situation when role is already created with some meaningful output.